spec/stylesheet.rb

```diff
@@ -249,7 +249,7 @@
       view.origin.x.should == 1
       view.frame = [[5,0],[5,5]]
       view.origin.x.should == 5
-      q.all.reapply_styles
+      q.reapply_styles
```

Comment:
David, this line here isn't clear to me. Are you try9ing to only apply the style to the root_view here?